dotCMS/src/main/java/com/dotcms/ai/app/AIModels.java

```diff
@@ -51,6 +51,7 @@ public class AIModels implements EventSubscriber<SystemTableUpdatedKeyEvent> {
     private static final int AI_MODELS_CACHE_TTL = 28800;  // 8 hours
     private static final int AI_MODELS_CACHE_SIZE = 256;
     private static final Lazy<AIModels> INSTANCE = Lazy.of(AIModels::new);
+    public static final String BEARER = "Bearer ";
```

Comment:
A lot of Http related values spread all over the code.
Probably it's a good idea to create a card to consolidate them in a single constants class.